front/components/EmptyCallToAction.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, PlusIcon } from "@dust-tt/sparkle";
+import Link from "next/link";
+import React, { ComponentType, MouseEvent } from "react";
+
+import { classNames } from "@app/lib/utils";
+
+export function EmptyCallToAction({
+  label,
+  disabled = false,
+  icon = PlusIcon,
+  href,
+  onClick,
+}: {
+  label: string;
+  disabled?: boolean;
+  icon?: ComponentType;
+  href?: string;
+  onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
+}) {
+  const button = (
+    <Button
+      disabled={disabled}
+      size="md"
```

Comment:
Should this be a prop?

Reply:
Right now we use the same everywhere so I think we can keep it a non prob until it is needed?